.travis.yml

```diff
@@ -22,6 +22,19 @@ jobs:
       cache: pip
       install: pip install moban
       script: .ci/check_moban.sh
+    - stage: coala
```

Comment:
and this added?

These changes are not related to the `SpaceConsistencyBear` issue afaik

Reply:
No your right these sections are not related. This is related to another issue https://github.com/coala/projects/pull/600. Actually makman2 and me were planning to fix the CI in this pr itself because gsoc is already upon us. I have discussed this with him and based on that appropriate action will be taken :)

Reply:
That's great. Just add that issue reference to the comment/pr as well.